src/views/Recorder/EmptyState.tsx

```diff
@@ -0,0 +1,96 @@
+import { useEffect, useRef } from 'react'
+import { css } from '@emotion/react'
+import { zodResolver } from '@hookform/resolvers/zod'
+import { DiscIcon } from '@radix-ui/react-icons'
+import { Button, Flex, Heading, Text, TextField } from '@radix-ui/themes'
+import { useForm } from 'react-hook-form'
+import { z } from 'zod'
+
+import { FieldGroup } from '@/components/Form'
+
+interface EmptyStateProps {
+  isLoading: boolean
+  onStart: (url?: string) => void
+}
+
+const RecorderEmptyStateSchema = z.object({
+  url: z.string(),
+})
+
+type RecorderEmptyStateFields = z.infer<typeof RecorderEmptyStateSchema>
+
+export function EmptyState({ isLoading, onStart }: EmptyStateProps) {
+  const inputRef = useRef<HTMLInputElement | null>(null)
+
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm<RecorderEmptyStateFields>({
+    resolver: zodResolver(RecorderEmptyStateSchema),
+    defaultValues: {
+      url: '',
+    },
+    shouldFocusError: false,
+  })
+
+  const { ref, ...inputProps } = register('url')
+
+  const onSubmit = ({ url }: RecorderEmptyStateFields) => {
+    onStart(url)
+  }
+
+  useEffect(() => {
+    window.requestAnimationFrame(() => {
+      inputRef.current?.focus()
+    })
+  }, [])
```

Comment:
Looks like we can get way with the good old `autoFocus` prop on the `TextField` in this case, since input is not added to dom conditionally. I did some testing and seemed to work as expected, but let me know if I missed something